src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```diff
@@ -352,13 +357,13 @@ def write_output_to_file(output_json: dict, output_file_path: Path) -> None:
             metadata_file_name = output_file_path.with_suffix(".json").name
             metadata_file_path = metadata_dir / metadata_file_name
             # Ensure the metadata directory exists
-            metadata_dir.mkdir(parents=True, exist_ok=True)
+            fs.mkdir(create_parents=True, path=metadata_dir, exist_ok=True)
```

Comment:
exists_ok is not supported by fs APIs. Pls take a look.

Reply:
Addressed